groups/validate.go

```diff
@@ -348,37 +417,47 @@ func TestHardcodedGroupsForParanoia(t *testing.T) {
 // and history of threads and also use web interface to operate the group)
 // More info:
 // 	https://developers.google.com/admin-sdk/groups-settings/v1/reference/groups#allowWebPosting
-func TestGroupsWhichShouldSupportHistory(t *testing.T) {
+func VerifyGroupsWhichShouldSupportHistory(config Config) {
```

Comment:
I'd love to make this more general to enforce that groups matching a certain pattern must have web history / web posting, but again **not in this PR**.

Reply:
Added TODO note